### ISSUE TITLE
release.sh needs to work in python 2 environments as well

### ISSUE THREAD

**Issue by rajivpb** [googledatalab/pydatalab#605]
Currently, urllib (InsecurePlatform errors, IIRC) issues prevent it from uploading to pypi, but these do not exist in python 3.

**Comment by rajivpb** [comment]
CC @chmeyers

**Comment by chmeyers** [comment]
Do these issues still exist after #585?

**Comment by rajivpb** [comment]
Yes. Looking through #585, I'm no longer sure why I mentioned it here. Appears to have been in error.
